src/sentry/web/urls.py

```diff
@@ -509,11 +514,6 @@
                     react_page_view,
                     name="sentry-organization-api-key-settings",
                 ),
-                url(
-                    r"^(?P<organization_slug>[\w_-]+)/auth/$",
-                    react_page_view,
-                    name="sentry-organization-auth-settings",
-                ),
                 url(
```

Comment:
Shouldn't this live in the new auth home as well?

Reply:
what do you mean?